shell/renderer/web_worker_observer.cc

```diff
@@ -15,28 +17,31 @@ namespace electron {
 
 namespace {
 
-static base::LazyInstance<
-    base::ThreadLocalPointer<WebWorkerObserver>>::DestructorAtExit lazy_tls =
-    LAZY_INSTANCE_INITIALIZER;
+static base::LazyInstance<base::ThreadLocalOwnedPointer<WebWorkerObserver>>::
+    DestructorAtExit lazy_tls = LAZY_INSTANCE_INITIALIZER;
```

Comment:
Can we change this to 
```suggestion
static base::NoDestructor<base::ThreadLocalOwnedPointer<WebWorkerObserver>> lazy_tls;
```